eZ/Bundle/EzPublishCoreBundle/Translation/Collector.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * File containing the Collector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Translation;
+
+/**
+ * Retrieves all installed ezplatform translation files.
+ */
+class Collector
```

Comment:
Instead of an internal tech name I would rather suggest VendorCollector cause it collector translation in vendor so this open the door to DbCollector.....

What do you think about it ?